nixos/modules/services/web-apps/pingvin-share.nix

```
@@ -0,0 +1,233 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.services.pingvin-share;
+  inherit (lib) mkOption mdDoc mkEnableOption mkIf mkPackageOptionMD types;
+in
+
+{
+  options = {
+    services.pingvin-share = {
+      enable = mkEnableOption (mdDoc "Pingvin Share, a self-hosted file sharing platform and an alternative for WeTransfer.");
```

```suggestion
      enable = mkEnableOption (mdDoc "Pingvin Share, a self-hosted file sharing platform");
```
the dot is already added by mkEnableOption. Also mentioning everywhere that the software is similar to WeTransfer is IMO unecessary and removes the own character of the software even if it very similar to wetransfer. We can keep it if you want, though.